marketplace_docs/templates/Fabric/scripts/01-initial-setup

```diff
@@ -35,3 +35,9 @@ rm -f /etc/zulip/zulip-secrets.conf /etc/zulip/settings.py
 #
 # For more details see https://chat.zulip.org/#narrow/stream/3-backend/topic/apt-daily
 systemctl disable apt-daily.timer apt-daily-upgrade.timer
+
+# Configure firewall. Else, img_check.sh will warn that ufw is not configufred.
```

Comment:
"configured" is mispelled; otherwise lgtm.

Reply:
Fixed.